ext/drreg/drreg.c

```diff
@@ -1609,9 +1609,11 @@ is_our_spill_or_restore(void *drcontext, instr_t *instr, bool *spill OUT,
                 opnd_get_disp(dr_reg_spill_slot_opnd(drcontext, SPILL_SLOT_1));
             uint DR_max_offs = opnd_get_disp(
                 dr_reg_spill_slot_opnd(drcontext, dr_max_opnd_accessible_spill_slot()));
-            if (DR_min_offs > DR_max_offs)
-                DR_min_offs = DR_max_offs;
-            slot = (offs - DR_min_offs) / sizeof(reg_t);
+            if (DR_min_offs > DR_max_offs) {
```

Comment:
commit message error: "but not restore" => "but no restore"

Reply:
Thanks